cms/djangoapps/contentstore/management/commands/export_olx.py

```diff
@@ -0,0 +1,113 @@
+"""
+A Django command that exports a course to a tar.gz file.
+
+If <filename> is '-', it pipes the file to stdout.
+
+This is used by Analytics research exports to provide researchers
+with course content.
+
+At present, it differs from Studio exports in several ways:
+
+* It does not include static content.
+* The top-level directory in the resulting tarball is a "safe"
+  (i.e. ascii) version of the course_key, rather than the word "course".
+* It only supports the export of courses.  It does not export libraries.
+
+"""
+
+import os
+import re
+import shutil
+import tarfile
+from tempfile import mktemp, mkdtemp
+from textwrap import dedent
+
+from path import Path as path
+
+from django.core.management.base import BaseCommand, CommandError
+
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore.xml_exporter import export_course_to_xml
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+
+
+class Command(BaseCommand):
+    """
+    Export a course to XML. The output is compressed as a tar.gz file
+
+    """
+    args = "<course_id> <output_filename>"
+    help = dedent(__doc__).strip()
+
+    def handle(self, *args, **options):
+        course_key, filename, pipe_results = self._parse_arguments(args)
+
+        export_course_to_tarfile(course_key, filename)
+
+        results = self._get_results(filename) if pipe_results else None
+
+        self.stdout.write(results, ending="")
+
+    def _parse_arguments(self, args):
+        """Parse command line arguments"""
+        try:
+            course_key = CourseKey.from_string(args[0])
+            filename = args[1]
+        except InvalidKeyError:
+            raise CommandError("Unparsable course_id")
+        except IndexError:
+            raise CommandError("Insufficient arguments")
+
+        # If filename is '-' save to a temp file
+        pipe_results = False
+        if filename == '-':
```

Comment:
Seems like the filename should explicitly be an optional parameter, maybe using `-o` and `--output`.